README.md

```diff
@@ -18,6 +18,23 @@ SPIR-V is defined by the Khronos Group Inc.
 See the [SPIR-V Registry][spirv-registry] for the SPIR-V specification,
 headers, and XML registry.
 
+## Verisoning SPIRV-Tools
+
+See [`CHANGES`](CHANGES) for a high level summary of recent changes, by version.
+
+SPIRV-Tools project version numbers are of the form `v`*year*`.`*index* and with
```

Comment:
I was following the "dotted integer" convention recommend for Perl modules:
https://metacpan.org/pod/CPAN::Meta::Spec#Version-Formats
It turns out that in Perl, tokens of the form  v2016.1  are a "v-string" that you can do sensible ordering comparisons on.

But I departed from the convention, since I added the "wip" to it to indicate work-in-progress.

I think I still prefer to keep the 'v'.